src/pages/Events.vue

```diff
@@ -18,7 +18,7 @@
             <div class="event-card--cta">October 25th-31st 2020</div>
           </div>
         </div>
-        <div class="event-card">
+        <div class="event-card" @click="redirectHackUTD">
```

Comment:
While you're already in the process of making changes to this page, I recommend wrapping both of the event cards with `a` tags instead of redirecting on click. This has a number of benefits including being better for accessibility (screen-readers, navigating via tab, etc.), SEO (by having a direct link), and user experience (knowing it's a link when hovering over the card.

Reply:
Should be all set now, thanks for the feedback!